2025-07-28 12:41:28,068 INFO    MainThread:7896 [wandb_setup.py:_flush():80] Current SDK version is 0.21.0
2025-07-28 12:41:28,068 INFO    MainThread:7896 [wandb_setup.py:_flush():80] Configure stats pid to 7896
2025-07-28 12:41:28,068 INFO    MainThread:7896 [wandb_setup.py:_flush():80] Loading settings from C:\Users\daotq\.config\wandb\settings
2025-07-28 12:41:28,069 INFO    MainThread:7896 [wandb_setup.py:_flush():80] Loading settings from C:\Users\daotq\Documents\Source_daotq\Git\Vehicle-Detection\wandb\settings
2025-07-28 12:41:28,069 INFO    MainThread:7896 [wandb_setup.py:_flush():80] Loading settings from environment variables
2025-07-28 12:41:28,069 INFO    MainThread:7896 [wandb_init.py:setup_run_log_directory():703] Logging user logs to C:\Users\daotq\Documents\Source_daotq\Git\Vehicle-Detection\wandb\run-20250728_124128-4g52o01a\logs\debug.log
2025-07-28 12:41:28,069 INFO    MainThread:7896 [wandb_init.py:setup_run_log_directory():704] Logging internal logs to C:\Users\daotq\Documents\Source_daotq\Git\Vehicle-Detection\wandb\run-20250728_124128-4g52o01a\logs\debug-internal.log
2025-07-28 12:41:28,069 INFO    MainThread:7896 [wandb_init.py:init():830] calling init triggers
2025-07-28 12:41:28,069 INFO    MainThread:7896 [wandb_init.py:init():835] wandb.init called with sweep_config: {}
config: {'weights': 'yolov5m.pt', 'cfg': '', 'data': 'dataset.yaml', 'hyp': {'lr0': 0.01, 'lrf': 0.2, 'momentum': 0.937, 'weight_decay': 0.0005, 'warmup_epochs': 3.0, 'warmup_momentum': 0.8, 'warmup_bias_lr': 0.1, 'box': 0.05, 'cls': 0.5, 'cls_pw': 1.0, 'obj': 1.0, 'obj_pw': 1.0, 'iou_t': 0.2, 'anchor_t': 4.0, 'fl_gamma': 0.0, 'hsv_h': 0.015, 'hsv_s': 0.7, 'hsv_v': 0.4, 'degrees': 0.0, 'translate': 0.1, 'scale': 0.5, 'shear': 0.0, 'perspective': 0.0, 'flipud': 0.0, 'fliplr': 0.5, 'mosaic': 1.0, 'mixup': 0.0, 'copy_paste': 0.0}, 'epochs': 50, 'batch_size': 16, 'imgsz': 640, 'rect': False, 'resume': False, 'nosave': False, 'noval': False, 'noautoanchor': False, 'evolve': None, 'bucket': '', 'cache': None, 'image_weights': False, 'device': '', 'multi_scale': False, 'single_cls': False, 'adam': False, 'sync_bn': False, 'workers': 8, 'project': 'runs/train', 'entity': None, 'name': 'exp', 'exist_ok': False, 'quad': False, 'linear_lr': False, 'label_smoothing': 0.0, 'upload_dataset': False, 'bbox_interval': -1, 'save_period': -1, 'artifact_alias': 'latest', 'local_rank': -1, 'freeze': 0, 'patience': 30, 'save_dir': 'runs\\train\\exp15', '_wandb': {}}
2025-07-28 12:41:28,070 INFO    MainThread:7896 [wandb_init.py:init():871] starting backend
2025-07-28 12:41:28,297 INFO    MainThread:7896 [wandb_init.py:init():874] sending inform_init request
2025-07-28 12:41:28,330 INFO    MainThread:7896 [wandb_init.py:init():882] backend started and connected
2025-07-28 12:41:28,333 INFO    MainThread:7896 [wandb_init.py:init():953] updated telemetry
2025-07-28 12:41:28,459 INFO    MainThread:7896 [wandb_init.py:init():977] communicating run to backend with 90.0 second timeout
2025-07-28 12:41:29,474 WARNING MainThread:7896 [wandb_init.py:init():1610] [no run ID] interrupted
Traceback (most recent call last):
  File "C:\Users\daotq\AppData\Local\Programs\Python\Python311\Lib\site-packages\wandb\sdk\wandb_init.py", line 1606, in init
    return wi.init(run_settings, run_config, run_printer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\daotq\AppData\Local\Programs\Python\Python311\Lib\site-packages\wandb\sdk\wandb_init.py", line 996, in init
    result = wait_with_progress(
             ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\daotq\AppData\Local\Programs\Python\Python311\Lib\site-packages\wandb\sdk\mailbox\wait_with_progress.py", line 24, in wait_with_progress
    return wait_all_with_progress(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\daotq\AppData\Local\Programs\Python\Python311\Lib\site-packages\wandb\sdk\mailbox\wait_with_progress.py", line 70, in wait_all_with_progress
    return _wait_handles(handle_list, timeout=progress_after)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\daotq\AppData\Local\Programs\Python\Python311\Lib\site-packages\wandb\sdk\mailbox\wait_with_progress.py", line 110, in _wait_handles
    results.append(handle.wait_or(timeout=remaining_timeout))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\daotq\AppData\Local\Programs\Python\Python311\Lib\site-packages\wandb\sdk\mailbox\mailbox_handle.py", line 118, in wait_or
    return self._fn(self._handle.wait_or(timeout=timeout))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\daotq\AppData\Local\Programs\Python\Python311\Lib\site-packages\wandb\sdk\mailbox\response_handle.py", line 84, in wait_or
    if not self._event.wait(timeout=timeout):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\daotq\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 629, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\daotq\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 331, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2025-07-28 12:41:30,316 INFO    MsgRouterThr:7896 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
